app/modules/import-export/module.php

```diff
@@ -86,6 +86,8 @@ public function __construct() {
 		if ( ElementorUtils::is_wp_cli() ) {
 			\WP_CLI::add_command( 'elementor kit', WP_CLI::class );
 		}
+
+		( new Usage() )->set_usage_data();
```

Comment:
```suggestion
		( new Usage() )->register();
```

See my comment in @mark-elementor's review comment

Reply:
Changed.
Mark is OK with both